py3status/modules/netdata.py

```diff
@@ -59,12 +59,27 @@ class Py3status:
     low_traffic = 400
     med_speed = 60
     med_traffic = 700
-    nic = 'eth0'
+    nic = None
 
     def __init__(self):
         self.old_transmitted = 0
         self.old_received = 0
 
+    def post_config_hook(self):
+        if self.nic is None:
+            # Read default gateway directly from /proc.
+            with open("/proc/net/route") as fh:
+                for line in fh:
+                    fields = line.strip().split()
+                    if fields[1] == '00000000' and int(fields[3], 16) & 2:
+                        self.nic = fields[0]
+                        break
+            self.py3.log('selected nic: %s' % self.nic)
+            if self.nic is None:
+                self.nic = 'lo'
+                self.py3.log('selected nic: %s' % self.nic)
```

Comment:
you only need to log this once.  for found nic is ok but when `lo` will log 
```
selected nic: None
selected nic: lo
```


a) less code to maintain
b) cleaner logs